modules/social_features/social_album/social_album.module

```diff
@@ -117,7 +117,7 @@ function social_album_preprocess_node(&$variables) {
 
   if (
     $node->bundle() === 'album' &&
-    $variables['view_mode'] === 'teaser' &&
+    in_array($variables['view_mode'], ['teaser', 'activity_comment', 'featured']) &&
```

Comment:
@kashandarash I am not sure if we need to have `featured` here. If so, then maybe we can just use image style `social_x_large` in the `embed_album_cover` view, the same as for other teasers. 

@OH17L Since you worked with this code, do you remember if we use embed_album_cover anywhere else except as for teasers? If so - why featured image style was choosen there?